playbooks/preflight-check.yml

```diff
@@ -91,12 +91,18 @@
   hosts: localhost
   gather_facts: false
   user: root
+  vars:
+    override_file: '/etc/openstack_delpoy/user_osa_variables_overrides.yml'
+    auth_key: 'glance_swift_store_auth_version'
+  set_fact:
+    cloudfiles_enabled: "{{ glance_swift_store_auth_address | match("*\.api\.rackspacecloud*") }}"
   tasks:
     - name: Set glance_swift_auth_version to 2 for CloudFiles
-      script: "scripts/glance_auth_version.py"
-      failed_when: script_response.rc not in [0,3]
-      changed_when: script_response.rc == 3
-      register: script_response
+      shell: >
```

Comment:
Might be able to use the lineinfile module instead. (http://docs.ansible.com/ansible/latest/lineinfile_module.html)